src/filterfunctions/sort.js

```diff
@@ -0,0 +1,9 @@
+const getSortedProducts = (data, state) => {
```

Comment:
use meaningful name instead of data and state.

Reply:
Filter functionality is working fine. Make above changes. Also check other's suggestion. It is good to merge.